src/pick.ts

```diff
@@ -1,23 +1,24 @@
-function get(object: unknown, path: Array<string>): unknown {
-  return path.reduce((obj: Record<string, unknown>, p: string) => {
-    return obj?.[p] as Record<string, unknown>
-  }, object as Record<string, unknown>)
+import type { StateTree } from 'pinia'
+
+function get(state: StateTree, path: string[]): unknown {
+  return path.reduce((obj, p) => {
+    return obj?.[p]
+  }, state)
 }
 
-function set(object: unknown, path: Array<string>, val: unknown): unknown {
+function set(state: StateTree, path: string[], val: unknown): StateTree {
```

Comment:
is there a reason for `string[]` over `Array<string>` ? I always preferred using generics

Reply:
It just looks shorter. I find that easier to read.

Reply:
I should have included the `array-type` rule to generic to my eslint-config 😄 
If we could stick to `Array<T>` syntax on the repo, I'd appreciate.